bin/GameIndex.dbf

```diff
@@ -1263,7 +1269,8 @@ Serial = SCES-50354
 Name   = Klonoa 2 - Lunatea's Veil
 Region = PAL-M5
 Compat = 5
-eeClampMode = 3		//Objects needed appear in wrong places without it
+eeClampMode = 3	//Objects needed appear in wrong places without it
```

Comment:
Trailing whitespace. It's not consistent with the others.

Reply:
It should be fixed now.

Reply:
It's still not fixed. Copy my line and replace it, that should fix it up.